rotkehlchen/tests/api/test_misc.py

```diff
@@ -349,6 +349,18 @@ def test_manage_ethereum_nodes(rotkehlchen_api_server):
         },
     )
     assert_proper_response_with_result(response)
+    response = requests.patch(
```

Comment:
What does this test for? That the request does not fail? Then add a comment.

Reply:
Yes. testing that the request doesn't fail as is an allowed change. Changing the etherscan optimism rpc node